src/styles/bloks/card.scss

```
@@ -0,0 +1,85 @@
+.card {
+  box-sizing: border-box;
+  width: $card_width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  
+  display: flex;
+  flex-direction: column;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin: 40px 16px 0;
+
+  }
+  &__product-code {
```

```suggestion
  }
  
  &__product-code {
```
Не забуваємо про відступи піж блоками